frontend/src/components/osh/OSHScansTable.tsx

```diff
@@ -117,6 +131,21 @@ const OSHScansTable = () => {
                     </a>
                   </strong>
                 </Td>
+                <Td dataLabel={columnNames.scanResults}>
+                  {scan.scan_results_url ? (
+                    <strong>
+                      <a
+                        href={scan.scan_results_url}
+                        target="_blank"
+                        rel="noreferrer"
+                      >
+                        results
+                      </a>
+                    </strong>
+                  ) : (
+                    <></>
+                  )}
```

Comment:
This should work and be more concise. Not tested
```suggestion
                  {scan.scan_results_url && (
                    <strong>
                      <a
                        href={scan.scan_results_url}
                        target="_blank"
                        rel="noreferrer"
                      >
                        results
                      </a>
                    </strong>
                  )}
```